src/src/Commands/InternalCommands.cpp

```diff
@@ -414,17 +414,17 @@ bool executeInternalCommand(command_case_data & data)
       COMMAND_CASE_A("posttohttp", Command_HTTP_PostToHTTP,  -1); // HTTP.h
       #endif // if FEATURE_POST_TO_HTTP
 #if FEATURE_CUSTOM_PROVISIONING
-      COMMAND_CASE_A(       "provisionconfig", Command_Provisioning_Config,       0); // Provisioning.h
-      #if FEATURE_ALTERNATIVE_CDN_URL
-      COMMAND_CASE_A( "provisioncustomcdnurl", Command_Provisioning_CustomCdnUrl, 0); // Provisioning.h
-      #endif // if FEATURE_ALTERNATIVE_CDN_URL
-      COMMAND_CASE_A(     "provisionsecurity", Command_Provisioning_Security,     0); // Provisioning.h
+      COMMAND_CASE_A( "provision", Command_Provisioning_Dispatcher, -1); // Provisioning.h
+      #ifdef PLUGIN_BUILD_MAX_ESP32 // FIXME DEPRECATED: Fallback for temporary backward compatibility
+      COMMAND_CASE_A(       "provisionconfig", Command_Provisioning_ConfigFallback,       0); // Provisioning.h
+      COMMAND_CASE_A(     "provisionsecurity", Command_Provisioning_SecurityFallback,     0); // Provisioning.h
```

Comment:
Why did you name this 'fallback' ?
It is the location of where to fetch the files/updates from.
It isn't a fallback like the CDN as there is no 'backdoor' in the firmware for fetching updates :)

Reply:
Ah now I get it.. fallback as in deprecated for older commands :)